frontend/app/dashboard/components/ScanReport.tsx

```
@@ -0,0 +1,233 @@
+                            onClick={() => setActiveTab(tab.id)}
+                            disabled={!hasOutput}
+                            className={`flex-shrink-0 px-5 py-3 flex items-center gap-2 text-sm font-medium transition-all border-b-2 ${isActive
+                                    ? 'border-current bg-white/5'
+                                    : 'border-transparent hover:bg-white/5'
+                                } ${hasOutput
+                                    ? 'text-white cursor-pointer'
+                                    : 'text-[#404050] cursor-not-allowed'
+                                }`}
+                            style={{ color: isActive ? tab.color : undefined }}
+                        >
+                            <span>{tab.icon}</span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)